src/interfaces/hybrid_crypto/structs.rs

```diff
@@ -0,0 +1,114 @@
+use std::marker::PhantomData;
```

Comment:
do we keep this file? (apparently not used anymore)

Reply:
No rigt. Same comment as before. Failed attempt at using opaque objects on the FFI. Removing it